play/speedy-math2/js/app.js

```diff
@@ -317,9 +342,67 @@ function getRandomNumber(min, max, type) {
 
 /**
  * Capitalizes the first letter of the word.
- * @param {String} str 
+ * @param {String} str
  * @returns {String} properCaptitalizedStr
  */
 function getProperCapitalization(str) {
     return str.charAt(0).toUpperCase() + str.slice(1);
-}
\ No newline at end of file
+}
+
+/**
+ * Display start screen, then initial countdown.
+ */
+function startScreen(){
+  toggleProblems();
+  startButtonEl.onclick = countdown;
+}
+
+/**
+ * Counts down from 3, then starts game
+ */
+function countdown(){
+    let count = 3;
+    startingEl.classList.toggle("hide");
+
+    const INTERVAL = setInterval( () => {
+
+        if (count < 1) {
+            clearInterval(INTERVAL);
+            countdownEl.classList.toggle("hide");
+
+            // show the UI
+            toggleProblems();
+            startTimer();
+        }
+
+        countdownEl.innerHTML = count;
+        count--;
+    }, 1000);
+}
+
+/**
+ * Counts down from 60, then displays result screen
+ */
+function startTimer(){
+    let timeRemaining = 60;
+
+    const INTERVAL = setInterval( () => {
+
+        if (timeRemaining <= 1) {
+            clearInterval(INTERVAL);
+        }
+
+          timeRemaining--;
+          timerEl.innerHTML = "Time: " + timeRemaining;
+    }, 1000);
+}
+
+/**
+ * Toggles the problem, timer, and score elements
+ */
+function toggleProblems(){
```

Comment:
Can you also make it focus on the input element so the user doesn't have to click on the input element when they enter their first answer?

Reply:
Changes made